libra_validation/fssh_based_2_state_models/analysis.py

```diff
@@ -124,12 +124,45 @@ def plot(mdls, methods):
             
                 plots(F"model_{mdl}_method_{method}", imdl, f["X"], f["Y"])
             
+
+def compute_accum_error(model_hdf_filename, exact_hdf_filename, istate_exact=0, istate_model=0):
+    """
+    This function computes the accumulate error between the population of the 
+    TSH/Ehrenfest and exact dynamics.
+    Args:
+        model_hdf_filename (string): The mem_data.hdf file name for (path).
+        model_hdf_filename (string): The data.hdf file name (path).
+        istate (integer): The initial state in the inital condition.
+    Returns:
+        acc_error (numpy arary): The average acumulation error.
+    """
+    # Open and read the model hdf file. This file name is mem_data.hdf but needs
+    # the full path to this file.
+    F = h5py.File(model_hdf_filename, 'r')
+    x1 = np.array(F['sh_pop_adi/data'])

```

Comment:
i believe, this copy is fine, just beware that F[...]  is no longer accessible once the file is closed